components/buttons/deploy.js

```diff
@@ -3,11 +3,24 @@ import Button from './button'
 
 const VERCEL_EXAMPLES_URL = 'github.com/vercel/vercel/tree/master/examples/'
 
-export default function DeployButton({ url }) {
+export default function DeployButton({ env, envDescription, envLink, url }) {
+  const formatEnv = () => {
+    const envListFormatted = env ? `&env=${env.toString()}` : ''
```

Comment:
I think you might need to check the length here and split the env vars via comma.

The `.filter(Boolean)` will also check if any of the array is an empty string.

```suggestion
    const envListFormatted = env.filter(Boolean).length >= 2 ? `&env=${env.join(',')}` : env[0] !== '' ? `&env=${env[0]}` : ''
```

Reply:
Ah yes, I knew I'd forgotten something, thanks!